pydarn/plotting/rtp.py

```diff
@@ -883,7 +881,7 @@ def plot_summary(cls, dmap_data: List[dict],
             Default: {'p_l': 'plasma',
                       'v': PyDARNColormaps.PYDARN_VELOCITY,
                       'w_l': PyDARNColormaps.PYDARN_VIRIDIS,
-                      'elv': PyDARNColormaps.PYDARN}
+                      'elv': 'inferno'}
```

Comment:
```suggestion
                      'elv': PyDARNColormaps.PYDARN_INFERNO}
```

Reply:
Thanks, plasma above it needed changing too!